src/styles/blocks/main.scss

```diff
@@ -0,0 +1,23 @@
+.main {
+  padding: 50px 40px;
+
+  &__products-grid {
+    display: grid;
+    grid-template-columns: repeat(4, auto);
+    justify-content: center;
+  }
```

Comment:
Good that you used grid!

But now you have one problem with card hight:
<img width="1029" alt="image" src="https://user-images.githubusercontent.com/99652389/178936495-288d93ed-e0ee-4965-a37a-5a713c380f56.png">

you can fix it by adding 1 style to grid instead of margins on cards.

Reply:
one more screenshot:
<img width="989" alt="image" src="https://user-images.githubusercontent.com/99652389/178933719-ce08dd03-2c29-4218-a2bb-5b6fb40ae1e7.png">

Reply:
@AndriiRut Thanks for your comment, I've fixed and now it's perfect.
I know about gaps, but in this task I was afraid that tests can fail so have added margins without manual checks.

Reply:
@sergii-nosachenko did you deploy new changes?

Reply:
@AndriiRut Forget, sorry. Now it should work.